src/components/Book.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = props => {
+  const { book } = props;
+  return (
+    <tr>
+      <td>{book.id}</td>
+      <td>{book.title}</td>
+      <td>{book.category}</td>
+    </tr>
+  );
+};
+
+Book.propTypes = {
+  book: PropTypes.objectOf(
+    PropTypes.oneOfType([
```

Comment:
- [ ] `oneOfType` is used to an object that could be one of many types. The `book` object it's always going to be the same for all books, therefore, we need to describe its type with a different approach here. There many good alternatives but one that fit our case the most it's `exact`.

```JS
Book.propTypes = {
  book: PropTypes.exact({
    id: PropTypes.number,
    title: PropTypes.string,
    category: PropTypes.string,
  }).isRequired,
};
```

You can learn more about Typechecking With PropTypes [here](https://reactjs.org/docs/typechecking-with-proptypes.html)